samples/bluetooth/iso_broadcast/src/main.c

```diff
@@ -28,12 +28,16 @@ static K_SEM_DEFINE(sem_iso_data, CONFIG_BT_ISO_TX_BUF_COUNT,
 #define INITIAL_TIMEOUT_COUNTER (BIG_TERMINATE_TIMEOUT_US / BIG_SDU_INTERVAL_US)
 
 static uint16_t seq_num;
+static uint32_t ts;
+static int32_t  delta;
```

Comment:
```suggestion
static int64_t  delta;
```

I think you need 64-bits for the delta, right? 
When the TS wraps from maximum to minimum then the delta will be nearly the maximum size of a uint32_t, which cannot be stored in a int32_t